openpgp/write.go

```diff
@@ -203,11 +203,18 @@ func Encrypt(ciphertext io.Writer, to []*Entity, signed *Entity, hints *FileHint
 		hashToHashId(crypto.SHA1),
 		hashToHashId(crypto.RIPEMD160),
 	}
-	// In the event that a recipient doesn't specify any supported ciphers
-	// or hash functions, these are the ones that we assume that every
-	// implementation supports.
-	defaultCiphers := candidateCiphers[len(candidateCiphers)-1:]
-	defaultHashes := candidateHashes[len(candidateHashes)-1:]
+
+	// If no preferences were specified, assume something safe and reasonable.
+	defaultCiphers := []uint8{
+		uint8(packet.CipherAES128),
+		uint8(packet.CipherAES256),
+	}
```

Comment:
Any reason not to add the [full list](https://github.com/golang/crypto/blob/c197bcf24cde29d3f73c7b4ac6fd41f4384e8af6/openpgp/packet/packet.go#L442-L446) by default? It might prevent a couple random bugs in the future.

	CipherAES256
	CipherAES192
	CipherAES128
	CipherCAST5
	Cipher3DES